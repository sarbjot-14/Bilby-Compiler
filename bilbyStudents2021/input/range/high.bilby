main{
	
    imm result := high <4 .. 7> ;
    print result $n;
    
    imm resultFloat := high <4.4 .. 7.4> ;
    
    print resultFloat $n;
    
    imm resultChar := high <#a .. #f> ;
    
    print resultChar $n;
	
}