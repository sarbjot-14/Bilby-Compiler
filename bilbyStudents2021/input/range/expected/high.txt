7
7.400000
f
